src/aequitas/plotting.py

```diff
@@ -0,0 +1,1063 @@
+import logging
+import math
+import re
+
+import numpy as np
+#import pandas as pd
+import collections
+import seaborn as sns
+import matplotlib.pyplot as plt
+import matplotlib.colors
+import matplotlib.cm
+
+
+from aequitas import squarify_flipped as sf
+
+logging.getLogger(__name__)
```

Comment:
I don't believe this statement on its own does anything -- (usually this would be used to assign a logger instance to a variable, but you're not doing so).

Reply:
I don't know that you have to fix all of these! And maybe I'm nuts, but I'm quite sure that the interface of the `logging` module is to use this method like so:

```suggestion
LOG = logging.getLogger(__name__)
```

Then you can use `LOG` the same as you're currently using the `logging` module; (I think the module top-level methods are really just linked to the root logger instance).

You don't need to have this line at all -- but the advantage is that logs generated from your module-specific `LOG` will reflect the module in which the logs were generated, (rather than all logs being sent to "root").